explainaboard/loaders/aspect_based_sentiment_classification.py

```diff
@@ -15,51 +20,31 @@ class AspectBasedSentimentClassificationLoader(Loader):
         please refer to `test_loaders.py`
     """
 
-    def __init__(self, source: Source, file_type: FileType, data: str = None):
-        if source is None:
-            source = Source.local_filesystem
-        if file_type is None:
-            file_type = FileType.tsv
-        self._source = source
-        self._file_type = file_type
-        self._data = data
-
-    def load(self) -> Iterable[Dict]:
-        """
-        aspect \t text \t label \t predicted_label
-        :return: class object
-        """
-        super().load()
-        data: List[Dict] = []
-        if self._file_type == FileType.tsv:
-            for id, dp in enumerate(self._raw_data):
-                aspect, text, true_label, predicted_label = dp[:4]
-                data.append(
-                    {
-                        "id": str(id),
-                        "aspect": aspect.strip(),
-                        "text": text.strip(),
-                        "true_label": true_label.strip(),
-                        "predicted_label": predicted_label.strip(),
-                    }
-                )
-        elif self._file_type == FileType.json:
-            for id, info in enumerate(self._raw_data):
-                aspect, text, true_label, predicted_label = (
-                    info["aspect"],
-                    info["text"],
-                    info["true_label"],
-                    info["predicted_label"],
-                )
-                data.append(
-                    {
-                        "id": str(id),
-                        "aspect": aspect.strip,
-                        "text": text.strip(),
-                        "true_label": true_label.strip(),
-                        "predicted_label": predicted_label.strip(),
-                    }
-                )
-        else:
-            raise NotImplementedError
-        return data
+    _default_file_type = FileType.tsv
+    field_names = ["aspect", "text", "true_label", "predicted_label"]
+    _default_file_loaders = {
+        FileType.tsv: TSVFileLoader(
+            [
+                FileLoaderField(0, field_names[0], FileLoaderDType.str, True),
```

Comment:
The first three fields here are pretty obvious from the context, but the final True is not. Could you use `strip_before_parsing=True` instead to make this more obvious?